BookingsApi/BookingsApi/Controllers/JusticeUserController.cs

```diff
@@ -54,8 +54,9 @@ public async Task<IActionResult> AddJusticeUser(AddJusticeUserRequest request)
                 ModelState.AddFluentValidationErrors(validation.Errors);
                 return ValidationProblem(ModelState);
             }       
+            int[] arrayInt = Array.ConvertAll(request.Role, value => (int) value);
```

Comment:
what does arrayInt represent? This would be a sign for a better named variable here

Reply:
changed